notebooks/249-oneformer-segmentation/249-oneformer-segmentation.ipynb

```diff
@@ -29,15 +29,19 @@
    "id": "39d6e067",
```

Comment:
<p>In bullet 2: models =&gt; model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Done